models/modelsmodule.cpp

```diff
@@ -176,6 +184,16 @@ ModelsModule::commandstring( void ) const
 void
 ModelsModule::init( SLIInterpreter* )
 {
+  kernel().model_manager.register_node_model< lin_rate_opn >( "lin_rate_opn" );
```

Comment:
Thanks for the suggestion. This issue of namings was discussed in the developer VC. We decided to go for abbreviated names as done in other neuron models.